src/components/definition-list/definition-list.spec.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { ThemeProvider } from 'styled-components';
+import mintTheme from '../../style/themes/mint';
+import { assertStyleMatch } from '../../__spec_helper__/test-utils';
+import { StyledDtDiv, StyledDdDiv } from './definition-list.style';
+import Dl from './dl.component';
+import Dt from './dt.component';
+import Dd from './dd.component';
+
+describe('DefinitionList', () => {
+  let wrapper;
+
+  describe('styles', () => {
+    it('matches the expected default styles of Dl', () => {
+      wrapper = (props, render = shallow) => {
```

Comment:
You repeat this pattern quite a lot, maybe it's worth extracting into a function to keep the tests DRY

Reply:
✅